environments/base/manifests/store-cache-statefulSet.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/component: store-cache
+    app.kubernetes.io/instance: observatorium-xyz
+    app.kubernetes.io/name: memcached
+    app.kubernetes.io/part-of: observatorium
+    app.kubernetes.io/version: 1.6.3-alpine
+  name: observatorium-xyz-thanos-store-memcached
+  namespace: observatorium
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: store-cache
+      app.kubernetes.io/instance: observatorium-xyz
+      app.kubernetes.io/name: memcached
+      app.kubernetes.io/part-of: observatorium
+  serviceName: observatorium-xyz-thanos-store-memcached
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: store-cache
+        app.kubernetes.io/instance: observatorium-xyz
+        app.kubernetes.io/name: memcached
+        app.kubernetes.io/part-of: observatorium
+        app.kubernetes.io/version: 1.6.3-alpine
+    spec:
+      containers:
+      - args:
+        - -m 1024
+        - -I 1m
+        - -c 1024
+        - -v
+        image: docker.io/memcached:1.6.3-alpine
+        name: memcached
+        ports:
+        - containerPort: 11211
+          name: http
```

Comment:
I don't think this is correct, iirc memcached speaks a binary protocol only on this port, am I mistaken?

Reply:
yea, it's RAW TCP (UDP available): https://github.com/memcached/memcached/blob/master/doc/protocol.txt we might want to rename it.

Reply:
True, my bad naming. 👍